homework/unique_ptr/test.cpp

```diff
@@ -59,6 +54,20 @@ TEST_F(UniquePointerTestSuite, passedWithMove)
     EXPECT_EQ(functionUptrAsArg(std::move(ptr)), m_someString);
 }
 
+TEST_F(UniquePointerTestSuite, moveOperator)
+{
+    my_unique_ptr<SomeUsefulClass> ptr2 = std::move(m_sut);
```

Comment:
this calls move constructor, not assignment operator. Assignment takes place when the object was already managing another memory. Do it in this way:
```suggestion
    my_unique_ptr<SomeUsefulClass> ptr2{new SomeUsefulClass{}};
    ptr2 = std::move(m_sut);
```